blog/forms.py

```diff
@@ -30,4 +30,13 @@ def clean(self, value):
 class BlogForm(ModelForm):
     class Meta:
         model = Entry
-        fields = ['pub_date', 'headline', 'content', 'ImageField']
\ No newline at end of file
+        fields = ['title', 'created_dttm', 'body', 'created_user']
+
+class UpdateForm(ModelForm):
```

Comment:
Name this something more meaningful.  If you have a bunch of different models you need to update and create forms for, they won't all be named "UpdateForm".  Instead, call it something like ``EntryUpdateForm``.